data/bangs.json

```diff
@@ -77777,10 +77777,10 @@
     "sc": "Online"
   },
   {
-    "s": "RECAP the law",
-    "d": "archive.recapthelaw.org",
+    "s": "CourtListener",
+    "d": "www.courtlistener.com",
     "t": "recap",
-    "u": "http://archive.recapthelaw.org/search/advanced/?q={{{s}}}&x=0&y=0&court=&docket_number=&case_name=&pacer_case_number=&date_filed=&date_filed_end=&date_terminated=&date_terminated_end=&date_last_filing=&date_last_filing_end=",
+    "u": "https://www.courtlistener.com/?type=r&q={{{s}}}&type=r&order_by=score%20desc",
```

Comment:
There are two of the `type=r` parameter. 

What do you think about adding another entry with this but with the `courtlistener` trigger (in addition to this change)?

Reply:
>There are two of the type=r parameter.

Oops, sorry. Will fix

>What do you think about adding another entry with this but with the courtlistener trigger (in addition to this change)?

As in keeping `recap` with `type=r` and also adding `courtlistener` with `type=o` (the "Case Law" tab)? Can do

Reply:
Oh apparently the courtlistener.com search form itself adds the duplicate `type`, removing the second one doesn't break anything (as expected) so I'll proceed with that change